app/models/manageiq/providers/openstack/cloud_manager/vm/operations/relocation.rb

```diff
@@ -0,0 +1,26 @@
+module ManageIQ::Providers::Openstack::CloudManager::Vm::Operations::Relocation
+  def raw_live_migrate(options = nil)
+    hostname         = options.try(:[], :hostname)
+    block_migration  = options.try(:[], :block_migration)  || false
+    disk_over_commit = options.try(:[], :disk_over_commit) || false
+    with_provider_connection do |connection|
+      connection.live_migrate_server(ems_ref, hostname, block_migration, disk_over_commit)
```

Comment:
I think errors will be handled elsewhere, or not? The UI should expose the API error, not sure how it's usually handled, by catching and returning error here? Or it will be caught on controller level?